frontend/resources/include/WindowManipulation.h

```diff
@@ -7,20 +7,23 @@
 
 #pragma once
 
+#include "GL_STUB.h"
+
 namespace megamol {
 namespace frontend_resources {
 
 struct WindowManipulation {
```

Comment:
The WindowManipulation should not be used anywhere without a GLFW WIndow. The need to stub it shows that it gets used (linked to) in some code path that is probably not cleanly separated from GL. Make use of the WindowManipulation resource optional and handle both cases everywhere it is used.